### PR TITLE
Add errorCallback on ref binding

### LINKED ISSUE #102: Allow listening to error callback
Is there a way to listen for `failureCallbackOrContext` function when binding to a reference?

Maybe add an `errorCallback` as the same as `readyCallback` which seems to be implemented here https://github.com/posva/vuexfire/blob/d041193beca9fbc82262230bf8a82c6a9e4082ab/src/index.js#L166-L168 ?

### PR DESCRIPTION
**Pull request by alex88**
Fixes #102

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -166,6 +167,9 @@ function bind ({
   if (readyCallback) {
     source.once('value', readyCallback)
   }
+  if (errorCallback) {
```
Comment (posva) [inline src/index.js]: can't this be done on the same if?

File: dist/vuexfire.js [hunk 2]
```diff
@@ -276,6 +276,7 @@ function bind (ref) {
   var ref_options = ref.options;
   var cancelCallback = ref_options.cancelCallback;
   var readyCallback = ref_options.readyCallback;
+  var errorCallback = ref_options.errorCallback;
```
Comment (posva) [inline dist/vuexfire.js]: Could you remove all files from dist please?

File: src/index.js (hunk 1 above)
Comment (alex88) [inline src/index.js]: What if you want just the error callback?

Comment (posva) [inline src/index.js]: passing `undefined` should still work, right?

Comment (alex88) [inline src/index.js]: Yeah, that would work, give me some time to finish some work and I'll change this

Comment (posva) [inline src/index.js]: There is no haste, don't worry 🙂

**Comment by alex88** [comment]
@posva any update on this?

**Comment by posva** [comment]
Sorry, I didn't see you added a commit. I'll release a new version with this soon. Thanks a lot for the contribution 🙂

**Comment by alex88** [comment]
Thank you for accepting it! :) No problem, I just thought there wasn't much to do after the commit

**Pull request merged**